CHANGELOG.asciidoc

```diff
@@ -40,7 +40,7 @@ https://github.com/elastic/beats/compare/v6.4.0...6.x[Check the HEAD diff]
 - Deregister pipeline loader callback when inputsRunner is stopped. {pull}[7893][7893]
 - Add backoff support to x-pack monitoring outputs. {issue}7966[7966]
 - Removed execute permissions systemd unit file. {pull}7873[7873]
-- Fix a race condition with the `add_host_metadata` and the event serialization. {pull}8223[8223]
+- Fix a race condition with the `add_host_metadata` and the event serialization. {pull}8223[8223] {pull}8653[8653]
```

Comment:
Double pull? What does it mean?

Reply:
This change is following another related fix (#8223), so both PRs are about the same issue.

Reply:
Thanks for the clarification :slightly_smiling_face: